### PR TITLE
Fix live image for imx8

### PR DESCRIPTION
**Pull request by nesteroff**

### CODE REVIEW

File: host/rootfs/etc/s6-linux-init/run-image/service/getty-ttyLP0/run [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/execlineb -P
+# SPDX-License-Identifier: EUPL-1.2+
+# SPDX-FileCopyrightText: 2020-2021 Alyssa Ross <hi@alyssa.is>
```
Comment (vilvo) [inline host/rootfs/etc/s6-linux-init/run-image/service/getty-ttyLP0/run:3]: Please use `# SPDX-FileCopyrightText: 2022 Unikie` instead for new files and add the same line for existing files with existing copyright when you are contributing new.

File: img/live/Makefile [hunk 2]
```diff
@@ -36,7 +36,7 @@ build/boot.fat: $(KERNEL) $(INITRAMFS) $(SYSTEMD_BOOT_EFI) build/spectrum.conf
 	$(MCOPY) -i $@ build/spectrum.conf ::/loader/entries
 	$(MCOPY) -i $@ $(KERNEL) ::/spectrum/linux
 	$(MCOPY) -i $@ $(INITRAMFS) ::/spectrum/initrd
-	$(MCOPY) -i $@ $(SYSTEMD_BOOT_EFI) ::/EFI/BOOT/BOOTX64.EFI
+	$(MCOPY) -i $@ $(SYSTEMD_BOOT_EFI) ::/EFI/BOOT/BOOTAA64.EFI
```
Comment (vilvo) [inline img/live/Makefile:40]: This is good to enable UEFI on aarch64 in this fork but more work is needed to upstream it.

File: host/rootfs/etc/s6-linux-init/run-image/service/getty-ttyLP0/run (hunk 1 above)
Comment (nesteroff) [reply host/rootfs/etc/s6-linux-init/run-image/service/getty-ttyLP0/run:3]: Ok, got it. It's fixed now.

**Comment by unbel13ver** [comment]
@nesteroff please make a rebase+force push instead of adding 3rd commit.

**Comment by vilvo** [comment]
> @nesteroff please make a rebase+force push instead of adding 3rd commit.

I'd rather rebase-squash.

**Review by unbel13ver — APPROVED** [review]

**Pull request merged**